metals/src/main/scala/scala/meta/internal/debug/protocol/OutputEventArguments.scala

```diff
@@ -0,0 +1,22 @@
+package scala.meta.internal.debug.protocol
+
+import org.eclipse.lsp4j.debug
+import org.eclipse.lsp4j.debug.OutputEventArgumentsCategory._
+
+object OutputEventArguments {
```

Comment:
Can we remove this class altogether and inline the `stdout` and `stderr` methods? They are only used once and the name `OutputEventArguments` conflicts with the lsp4j name.